### PR TITLE
Fix/issue #13806

### LINKED ISSUE #13806: New Collectible Details tabs
**There are also bespoke tabs inside the collection detail page depending on the type of collectible:**

Each community collectible features the tabs:
- Properties, Activity, Permissions - permissions tab to be designed later

Each collectible from a collection features the tabs:
- Traits, Activity, Links
(Note the new links tab designs here:
https://www.figma.com/file/FkFClTCYKf83RJWoifWgoX/Wallet-v2?type=design&node-id=21257-32122&mode=design&t=z76IWrH4Ov5fvbfc-4)

Each 1 of 1 collectible (not belonging to a community or collection) features the tabs:
- Activity, Links

### PR DESCRIPTION
**Pull request by Khushboo-dev-cpp**
fixes #13806 

go pr : https://github.com/status-im/status-go/pull/5007

### What does the PR do

Implements the collectible details tabs as oer new design. There is an issue with activity not showing up which will be addresses here https://github.com/status-im/status-desktop/issues/14256

### Affected areas

CollectibleDetailsView

### StatusQ checklist

- [ ] add documentation if necessary (new component, new feature)
- [ ] update sandbox app
  - in case of new component, add new component page
  - in case of new features, add variation to existing component page
  - nice to have: add it to the demo application as well
- [ ] test changes in both light and dark theme?

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

https://github.com/status-im/status-desktop/assets/60327365/dfdafaf5-2a19-451c-bb42-fe4fcb7f0dff

### CODE REVIEW

File: src/app/modules/shared_modules/collectible_details/controller.nim [hunk 1]
```diff
@@ -85,6 +85,11 @@ QtObject:
         if self.detailedEntry.updateDataIfSameID(collectible):
           break
 
+  proc processGetCollectibleSocialsResponse(self: Controller, response: JsonNode) =
+    let res = fromJson(response, backend_collectibles.CollectibleSocialsMessage)
+    self.detailedEntry.updateDataIfSameID(res)
+    self.detailedEntryChanged()
```
Comment (stefandunca) [inline src/app/modules/shared_modules/collectible_details/controller.nim:91]: The `updateDataIfSameID` might not do any change, does it make sense to trigger the signal `detailedEntryChanged`?

File: ui/app/AppLayouts/Wallet/views/collectibles/CollectibleDetailView.qml [hunk 2]
```diff
@@ -139,7 +148,7 @@ Item {
                             property int modelIndex: index
                             anchors.top: parent.top
                             anchors.left: parent.left
-                            sourceComponent: root.isCommunityCollectible && (root.isOwnerTokenType || root.isTMasterTokenType) ? privilegedCollectibleImage: collectibleimage
+                            sourceComponent: isCollectibleLoading ? collectibleimage: root.isCommunityCollectible && (root.isOwnerTokenType || root.isTMasterTokenType) ? privilegedCollectibleImage: collectibleimage
```
Comment (stefandunca) [inline ui/app/AppLayouts/Wallet/views/collectibles/CollectibleDetailView.qml:151]: Really complex binding. How about multi-line formatting or use `if`/`else`
e.g.
```qml
sourceComponent: isCollectibleLoading
                       ? collectibleimage
                       : root.isCommunityCollectible && (root.isOwnerTokenType || root.isTMasterTokenType)
                           ? privilegedCollectibleImage
                           : collectibleimage
```

**Review by stefandunca — APPROVED** [review]
Please see comments inline

File: src/app/modules/shared_models/collectibles_entry.nim [hunk 3]
```diff
@@ -30,6 +30,8 @@ QtObject:
       generatedId: string
       generatedCollectionId: string
       tokenType: TokenType
+      website: string
```
Comment (dlipicar) [inline src/app/modules/shared_models/collectibles_entry.nim:33]: do we really need separate variables? we could use/update the `data` itself

File: src/app/modules/shared_modules/collectible_details/controller.nim (hunk 1 above)
Comment (Khushboo-dev-cpp) [reply src/app/modules/shared_modules/collectible_details/controller.nim:91]: make sense! will remove it as events for dataChanged will be emitted from within the entry as it is

File: src/app/modules/shared_models/collectibles_entry.nim (hunk 3 above)
Comment (Khushboo-dev-cpp) [reply src/app/modules/shared_models/collectibles_entry.nim:33]: sure can do!

**Pull request merged**